src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -525,8 +531,56 @@ export const OscapList = () => {
         <TextListItem component={TextListItemVariants.dd}>
           {oscapProfile}
         </TextListItem>
+        <TextListItem
```

Comment:
Could we somehow use the `<OscapProfileInformation>` component here so we don't have two duplicate lists we'd have to keep in mind when making changes to them?